README.md

```diff
@@ -2,7 +2,7 @@
 
 [![DOI](https://zenodo.org/badge/125840716.svg)](https://zenodo.org/badge/latestdoi/125840716)
 [![Snakemake](https://img.shields.io/badge/snakemake-≥4.8.1-brightgreen.svg)](https://snakemake.bitbucket.io)
-[![CircleCI](https://circleci.com/gh/ctmrbio/stag-mwc/tree/master.svg?style=svg)](https://circleci.com/gh/ctmrbio/stag-mwc/tree/master)
+[![CircleCI](https://circleci.com/gh/ctmrbio/stag-mwc/tree/develop.svg?style=svg)](https://circleci.com/gh/ctmrbio/stag-mwc/tree/develop)
```

Comment:
When emerged, this will need to be updated. Minor, but there

Reply:
Good call! 
I had a feeling I had forgotten about something, thanks for noticing!